src/SSEConnection.js

```diff
@@ -0,0 +1,54 @@
+import AbstractConnection from "./AbstractConnection";
+import ConnectionEvent from "./ConnectionEvent";
+
+/**
+ * Server Sent Event Connections
+ */
+export default class SSEConnection extends AbstractConnection {
+  constructor(url, options = {}) {
+    super(url);
+    this.options = options;
```

Comment:
please define these with `Object.definePropert(y|ies)`, see `XHRConnection` for reference 👍 

We want them to be protected or private, where possible.